src/schema/rules/sidecars/pet.yaml

```diff
@@ -181,3 +181,22 @@ PETReconstruction:
             level: recommended
         SinglesRate:
             level: recommended
+
+# PET task information
+PETTask:
+    selectors:
+    - modality == "pet"
```

Comment:
Should `"task" in entities` (or whatever the appropriate syntax is) be included here? Essentially, is this task-related metadata recommended when there is no task entity in the filename?

Reply:
ah yeah you are right: those metadata are conditional on task entity being present.
let me check how this expressed in the schema.

Reply:
fixed.